enterprise/server/webhooks/webhook_data/webhook_data.go

```diff
@@ -17,6 +17,10 @@ func init() {
 type WebhookData struct {
 	// EventName is the canonical event name that this data was created from.
 	EventName string
+	// PushedBranch is the name of the branch in the source repo that triggered the
+	// event when pushed. Note that for forks, the branch here references the branch
+	// name in the forked repository.
```

Comment:
Could we make this clearer with an example, like:

Normal case:
 - PushedBranch: "report_statuses"
 - TargetBranch: "report_statuses"

Forked repo case:
 - PushedBranch: "report_statuses_fork"
 - TargetBranch: "master"